app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -159,6 +159,7 @@ def pre_prov
 
   def render_updated_templates
     report_scopes = [:eligible_for_provisioning]
+    report_scopes.push([:search, params[:search_text]])
```

Comment:
`:search` is liked to the change mentioned here - https://github.com/ManageIQ/manageiq/blob/11349b26997691ab8600bf4c927a743dd110f475/app/models/miq_template.rb#L35